data/menus.yaml

```diff
@@ -18,14 +27,12 @@
     - leanproject: leanproject.html
     - How pieces fit together: toolchain.html
     - About MWEs: mwe.html
-    - API documentation: https://leanprover-community.github.io/mathlib_docs
 
 - title: Library overviews
   items:
     - Library overview: mathlib-overview.html
-    - Wiedijk's 100 theorems: 100.html
     - Undergraduate maths: undergrad.html
-    - Other Lean projects: lean_projects.html
+    - Wiedijk's 100 theorems: 100.html
 
 - title: Theory docs
```

Comment:
I'm not convinced this needs to be on the side panel. These are super out of date. (The p-adics one is my fault.) In fact, I think there's an argument for removing them from the site entirely, until we have a procedure for making sure they stay updated, but that's another discussion.

Reply:
The side panel should be a complete list of what's on the web site; otherwise people get lost and can't find the page they're looking for.

My secret hope is that making this pages more visible will motivate other people to clean them up (or maybe turn them into module doc strings).

Reply:
Hmm. I'm just going to delete the p-adics one now. It predates module docs and is subsumed already. Looking at them all again, I can see arguments for keeping especially the natural numbers, sets, and topology writeups. LA and CT seem outdated.